streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter.java

```diff
@@ -842,12 +830,6 @@ public Options setSstFileManager(final SstFileManager sstFileManager) {
         return this;
     }
 
-    @Override
-    public Options setLogger(final org.rocksdb.Logger logger) {
-        dbOptions.setLogger(logger);
```

Comment:
If this was removed, too, can you add it to the PR description.

Reply:
@mjsax thanks for feedback. I wanted to confirm setLogger() method. Parameter inside setLogger has changed to LoggerInterface. I've made corrections and put method back in that java class.